snapcraft/cli/_options.py

```diff
@@ -61,6 +62,23 @@ def _add_build_options(func):
     return _add_build_options
 
 
+def get_build_environment(**kwargs):
+    force_use_lxd = kwargs.get("use_lxd")
+    force_destructive_mode = kwargs.get("destructive_mode")
```

Comment:
This will get uglier if we add more `--use-<provider>`, but we can productize that later when the new options are added (if ever).